app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/manualcodeentry/ManualCodeEntryScreen.kt

```diff
@@ -126,7 +127,8 @@ fun ManualCodeEntryScreen(
             Text(
                 text = stringResource(id = R.string.enter_key_manually),
                 style = BitwardenTheme.typography.titleMedium,
-                modifier = Modifier.padding(horizontal = 16.dp),
+                modifier = Modifier.padding(horizontal = 16.dp)
+                    .testTag("EnterKeyManuallyButton"),
```

Comment:
Please run the formatter

Reply:
This is still incorrect, it should be:

```kotlin
                modifier = Modifier
                    .padding(horizontal = 16.dp)
                    .testTag("EnterKeyManuallyButton"),
```

There is also a tutorial for setting up a macro for this in the [readme](https://github.com/bitwarden/android/blob/main/README.md#setup).